openandroidinstaller/assets/configs/dreamlte.yaml

```diff
@@ -0,0 +1,24 @@
+metadata:
+  maintainer: Allaeddine BOULEFAAT (allaeddineomc)
+  device_name: Samsung Galaxy s 8
+  is_ab_device: false
+  device_code: dreamlte 
+  supported_device_codes:
+    - dreamlte 
```

Comment:
It would also be nice to have the user checked, that their device is updated to the latest version. Please consider adding the `requirements` category and the `android` field with the latest version available. (It should be 9.0 or something like that.)